build-tools/packages/version-tools/src/test/schemes.test.ts

```diff
@@ -90,6 +90,12 @@ describe("detectVersionScheme", () => {
         assert.strictEqual(detectVersionScheme(input), expected);
     });
 
+    it("detects 0.0.0-105091-test is semver", () => {
+        const input = `0.0.0-105091-test`;
+        const expected = "semver";
+        assert.strictEqual(detectVersionScheme(input), expected);
+    });
+
```

Comment:
This test passes as expected with no code change. The bug was in the command logic.